.github/workflows/cron.yml

```diff
@@ -0,0 +1,338 @@
+on:
```

Comment:
Some of this logic I was thinking of extracting into bash functions or moving directly into the cli tool. I can do later, if needed.

Reply:
DO NOT REVIEW this file.
I'm currently working on splitting it.